crates/ruff/src/registry.rs

```diff
@@ -191,6 +191,9 @@ pub enum Linter {
     /// [Airflow](https://pypi.org/project/apache-airflow/)
     #[prefix = "AIR"]
     Airflow,
+    /// [Perflint](https://pypi.org/project/perflint/)
+    #[prefix = "W8"]
```

Comment:
I'm not sure about the prefix here as we would want a 3 letter abbreviation for the plugin name (`PER`, `PRF`?). It's ok to deviate from the actual rule code for consistency :)

\cc @charliermarsh

Reply:
Maybe `PERF`? We've started to use some four-letter prefixes.